extension/src/test-utils.ts

```diff
@@ -0,0 +1,58 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import { Context, QueryResult, SqlClient } from '@exasol/extension-manager-interface';
+import { ExaScriptsRow } from '@exasol/extension-manager-interface/dist/exasolSchema';
+import * as jestMock from "jest-mock";
+
+const EXTENSION_SCHEMA_NAME = "ext-schema"
+
+export function getInstalledExtension(): any {
+    // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
+    return (global as any).installedExtension
```

Comment:
where is global from?

Reply:
This is the "global" JS object. We use it to [register an extension](https://github.com/exasol/extension-manager-interface/blob/main/src/api.ts#L199-L206) to make it available to the Extension Manager's JS runtime.